File: Python FastApi/models/message.py
from pydantic import BaseModel, Field, EmailStr
from datetime import datetime
from bson import ObjectId
from typing import Optional

class MessageBase(BaseModel):
    name: str
    surname: str
    email: EmailStr
    phone: int
    message: str
    isread: bool = False

class MessageCreate(MessageBase):
    pass

class MessageInDB(MessageBase):
    id: str = Field(..., alias="_id")
    created_at: datetime

    class Config:
        populate_by_name = True
        json_encoders = {ObjectId: str}

class MessageResponse(MessageInDB):
    pass

File: .Net/WhoEstate.API/Entities/Message.cs
using MongoDB.Bson;
using MongoDB.Bson.Serialization.Attributes;
using System;

namespace WhoEstate.API.Entities
{
    public class Message
    {
        [BsonId]
        [BsonRepresentation(BsonType.ObjectId)]
        public string Id { get; set; }

        [BsonElement("name")]
        public string Name { get; set; }

        [BsonElement("surname")]
        public string Surname { get; set; }

        [BsonElement("email")]
        public string Email { get; set; }

        [BsonElement("phone")]
        public long Phone { get; set; }

        [BsonElement("message")]
        public string MessageText { get; set; }

        [BsonElement("isread")]
        public bool IsRead { get; set; } = false;

        [BsonElement("createdAt")]
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
    }
}